<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open &amp;Recent</string>
     </property>
     <addaction name="actionOpen_All"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Cut"/>
    <addaction name="separator"/>
    <addaction name="action_Find_Replace"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="separator"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="action_Cut"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionentity"/>
   <addaction name="actionAction"/>
   <addaction name="actionStatus"/>
   <addaction name="actionBackDrop"/>
   <addaction name="actioncondition"/>
   <addaction name="actionApply"/>
   <addaction name="actionTool"/>
   <addaction name="actionNode_Tree"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normaloff>resources/file-new.png</normaloff>resources/file-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="toolTip">
    <string>Create a New Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normaloff>resources/file-open.png</normaloff>resources/file-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>resources/file-save.png</normaloff>resources/file-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the Current Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset>
     <normaloff>resources/file-exit.png</normaloff>resources/file-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset>
     <normaloff>resources/edit-copy.png</normaloff>resources/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy Slected Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset>
     <normaloff>resources/edit-paste.png</normaloff>resources/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste Copied Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset>
     <normaloff>resources/edit-cut.png</normaloff>resources/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;ut</string>
   </property>
   <property name="toolTip">
    <string>Cut Selected Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionOpen_All">
   <property name="text">
    <string>Open All</string>
   </property>
   <property name="toolTip">
    <string>Open All Recent Documents</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset>
     <normaloff>resources/help-content.png</normaloff>resources/help-content.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_Find_Replace">
   <property name="text">
    <string>&amp;Find and Replace...</string>
   </property>
   <property name="toolTip">
    <string>Launch Find and Replace Dialog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionentity">
   <property name="icon">
    <iconset>
     <normaloff>resources/entity.png</normaloff>resources/entity.png</iconset>
   </property>
   <property name="text">
    <string>entity</string>
   </property>
   <property name="toolTip">
    <string>Create an Entity</string>
   </property>
  </action>
  <action name="actionAction">
   <property name="icon">
    <iconset>
     <normaloff>resources/action.png</normaloff>resources/action.png</iconset>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
   <property name="toolTip">
    <string>Create an Action</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="icon">
    <iconset>
     <normaloff>resources/status.png</normaloff>resources/status.png</iconset>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
   <property name="toolTip">
    <string>Create a Status</string>
   </property>
  </action>
  <action name="actionBackDrop">
   <property name="icon">
    <iconset>
     <normaloff>resources/BackDrop.png</normaloff>resources/BackDrop.png</iconset>
   </property>
   <property name="text">
    <string>BackDrop</string>
   </property>
   <property name="toolTip">
    <string>Create a Group</string>
   </property>
  </action>
  <action name="actionApply">
   <property name="icon">
    <iconset>
     <normaloff>resources/Apply.png</normaloff>resources/Apply.png</iconset>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="toolTip">
    <string>Apply an action to cheked entities</string>
   </property>
  </action>
  <action name="actioncondition">
   <property name="icon">
    <iconset>
     <normaloff>resources/condition.png</normaloff>resources/condition.png</iconset>
   </property>
   <property name="text">
    <string>condition</string>
   </property>
   <property name="toolTip">
    <string>Create a new condition</string>
   </property>
  </action>
  <action name="actionNode_Tree">
   <property name="icon">
    <iconset>
     <normaloff>resources/Tree.png</normaloff>resources/Tree.png</iconset>
   </property>
   <property name="text">
    <string>Node Tree</string>
   </property>
   <property name="toolTip">
    <string>Display Node Tree</string>
   </property>
  </action>
  <action name="actionTool">
   <property name="icon">
    <iconset>
     <normaloff>resources/tool.png</normaloff>resources/tool.png</iconset>
   </property>
   <property name="text">
    <string>Tool</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
